sysctl.d/50-default.conf

```diff
@@ -36,3 +36,7 @@ net.core.default_qdisc = fq_codel
 # Enable hard and soft link protection
 fs.protected_hardlinks = 1
 fs.protected_symlinks = 1
+
+# Enable regular file and FIFO protection
```

Comment:
Not sure if the blank line and separate comment are necessary or if all of `fs.protected_*` should just be one block, with the L36 comment adjusted accordingly.